app/src/main/java/com/duckduckgo/app/browser/aura/AuraExperimentManagerImpl.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2024 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.browser.aura
+
+import com.duckduckgo.app.aura.AuraExperimentManager
+import com.duckduckgo.app.referral.AppReferrerDataStore
+import com.duckduckgo.app.statistics.store.StatisticsDataStore
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.installation.api.installer.InstallSourceExtractor
+import com.squareup.anvil.annotations.ContributesBinding
+import dagger.SingleInstanceIn
+import javax.inject.Inject
+
+@ContributesBinding(AppScope::class)
+@SingleInstanceIn(AppScope::class)
+class AuraExperimentManagerImpl @Inject constructor(
+    private val auraExperimentFeature: AuraExperimentFeature,
+    private val auraExperimentListJsonParser: AuraExperimentListJsonParser,
+    private val installSourceExtractor: InstallSourceExtractor,
+    private val statisticsDataStore: StatisticsDataStore,
+    private val appReferrerDataStore: AppReferrerDataStore,
+) : AuraExperimentManager {
+
+    override suspend fun initialize() {
+        if (auraExperimentFeature.self().isEnabled()) {
+            installSourceExtractor.extract()?.let { source ->
+                val settings = auraExperimentFeature.self().getSettings()
+                val packages = auraExperimentListJsonParser.parseJson(settings).list
+                if (packages.contains(source)) {
+                    if (statisticsDataStore.variant == RETURNING_USER) {
+                        appReferrerDataStore.returningUser = true
```

Comment:
Said in another place, but why do we need to re-store a returning user flag? it also seems that is used as follows

```kotlin
val returningUser = statisticsDataStore.variant == RETURNING_USER_VARIANT || appReferrerDataStore.returningUser
```
and fori t to be set to true `statisticsDataStore.variant == RETURNING_USER_VARIANT` needs to be `true` so I don't think we need two sources of truth for this